src/NServiceBus.Gateway.Tests/packages.config

```diff
@@ -4,6 +4,6 @@
   <package id="ApprovalTests" version="3.0.11" targetFramework="net452" />
   <package id="ApprovalUtilities" version="3.0.11" targetFramework="net452" />
   <package id="Mono.Cecil" version="0.9.6.1" targetFramework="net452" />
-  <package id="NServiceBus" version="6.0.0" targetFramework="net452" />
```

Comment:
> do you see any issues?

not directly, it's the same with the acceptance tests anyway so I guess it doesn't really matter.